kalite/contentload/management/commands/unpack_assessment_zip.py

```diff
@@ -61,9 +63,19 @@ def handle(self, *args, **kwargs):
         else:                   # file; just open it normally
             f = open(ziplocation, "rb")
 
+        def unpack():
+            zf = zipfile.ZipFile(f, "r")
+            unpack_zipfile_to_content_folder(zf)
+
+        unpack_thread = threading.Thread(target=unpack)
+        unpack_thread.daemon = True
+        unpack_thread.start()
         logging.info("Unpacking...")
-        zf = zipfile.ZipFile(f, "r")
-        unpack_zipfile_to_content_folder(zf)
+        while unpack_thread.is_alive():
```

Comment:
I see. So the main purpose of the changes here is to show to the user that something is still happening.